src/contributors/VaasuCodez.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Vaasu", 
+    "desc": "Student",
+    "github": "https://github.com/VaasuCodez",
```

Comment:
```suggestion
    "github": "https://github.com/VaasuCodez"
```
Remove this trailing comma or else add your twitter or linkedin profile.

Reply:
Thanks a bunch!